wagtail_storages/__init__.py

```diff
@@ -1,3 +1,3 @@
-__version__ = "0.5.1"
+__version__ = "0.6rc1"
```

Comment:
nitpick: it the maintainer's decision on what to do with the version

Reply:
OK thanks